rollup.config.js

```diff
@@ -36,6 +36,7 @@ const config = {
     nodeResolve({mainField: ['jsnext:main']}),
   ].concat(compress ? terser() : []),
   output,
+  inlineDynamicImports: true,
```

Comment:
I believe this requires dynamic imports to be imported before continuing to create the bundle.  It won't effect the production code and this still bundles in less than 1 second.  I think this is well worth the tradeoff to keep as much compatibility as possible.

Alternatively, we could switch from UMD to something like AMD (I read [this](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm) to learn about the differences). This would change things so that our code would compile to more front end specific directives and continues to be bundle in about 1/2 a second. I admit that our code is simple enough I have a hard time believing even with AMD, we'd have compatibility issues with server side executed code.